doubletdetection/doubletdetection.py

```diff
@@ -119,15 +133,28 @@ class BoostClassifier:
             doublet.
     """
 
-    def __init__(self, boost_rate=0.25, n_components=30, n_top_var_genes=10000, new_lib_as=None,
-                 replace=False, phenograph_parameters={'prune': True}, n_iters=25,
-                 normalizer=None, random_state=None):
+    def __init__(
+        self,
+        boost_rate=0.25,
+        n_components=30,
+        n_top_var_genes=10000,
+        replace=False,
+        use_phenograph=True,
```

Comment:
we could make use_phenograph False by default if we are not requiring it for install